bucket4j/src/main/java/com/linecorp/armeria/server/throttling/tokenbucket/package-info.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Rate-limiting throttling implementation based on
+ * <a href="https://en.wikipedia.org/wiki/Token_bucket">Token-Bucket</a> algorithm
+ * and <a href="https://github.com/vladimir-bukhtoyarov/bucket4j">Bucket4j</a> library.
+ *
+ */
+@NonNullByDefault
+package com.linecorp.armeria.server.throttling.tokenbucket;
```

Comment:
How about `bucket4j` instead of `tokenbucket`?

Reply:
I think we've talked about it in the past. "Bucket4j" is a library. It provides an implementation of "TokenBucket" algorithm. "TokenBucket" algorithm is what used for throttling and that's what is important. The Armeria's new module already called "bucket4j" that links it to the library. I think it's sufficient. I prefer to keep `tokenbucket` in the package name.

Reply:
I'd prefer having bucket4j in the package name because in the future we may want to add another module that provides similar functionality based on a other library than bucket4j then we will have a package name conflict.

Reply:
Renamed, as you insist.

Reply:
Thanks!